### PR TITLE
testing/moosefs: new aport

### PR DESCRIPTION
**Pull request by uu**
https://moosefs.com/
based on https://github.com/alpinelinux/aports/tree/master/testing/lizardfs
requested on https://gitlab.alpinelinux.org/alpine/aports/issues/2531

### CODE REVIEW

File: testing/moosefs/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+		--infodir=/usr/share/info \
+		--datadir=/usr/share \
+		--localstatedir=/var/lib \
+		--libdir=/usr/lib64
+	make
+}
+
+prepare() {
+	autoreconf -fi
+}
```
Comment (Cogitri) [inline testing/moosefs/APKBUILD]: nitpick, but defining prepare before build makes more sense to me

File: testing/moosefs/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+		--sysconfdir=/etc \
+		--with-default-user=mfs \
+		--with-default-group=mfs \
+		--disable-dependency-tracking \
+		--disable-silent-rules \
+		--mandir=/usr/share/man \
+		--infodir=/usr/share/info \
+		--datadir=/usr/share \
+		--localstatedir=/var/lib \
+		--libdir=/usr/lib64
```
Comment (Cogitri) [inline testing/moosefs/APKBUILD]: I think this should be /usr/lib

File: testing/moosefs/APKBUILD [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+pkgver="3.0.105"
+pkgrel=0
+pkgdesc="Open Source, Petabyte, Fault-Tolerant, Highly Performing, Scalable Network Distributed File System"
+url="https://moosefs.com"
+arch="all"
+license="GPL-2.0-only"
+makedepends="autoconf automake zlib-dev libpcap-dev libtool fuse-dev python3-dev"
+pkgusers="mfs"
+pkggroups="mfs"
+subpackages="$pkgname-doc $pkgname-client $pkgname-master $pkgname-chunkserver $pkgname-metalogger $pkgname-cgi::noarch $pkgname-cgiserv::noarch"
```
Comment (Cogitri) [inline testing/moosefs/APKBUILD]: Please break this line after 80 lines

Also, I guess this should come with a $pkgname-openrc subpackage.

Comment (uu) [reply testing/moosefs/APKBUILD]: >Please break this line after 80 lines

done

>Also, I guess this should come with a $pkgname-openrc subpackage.

inits belongs to subpackages and I dunno how to add them. I appreciate if you could show how achieve that.

File: testing/moosefs/APKBUILD (hunk 2 above)
Comment (uu) [reply testing/moosefs/APKBUILD]: done

File: testing/moosefs/APKBUILD (hunk 1 above)
Comment (uu) [reply testing/moosefs/APKBUILD]: moved:)

File: testing/moosefs/APKBUILD (hunk 3 above)
Comment (Cogitri) [reply testing/moosefs/APKBUILD]: Hm, not sure how we handle initd files of subpackages to be honest :thinking: 

Do you know more, @maxice8 ?

Comment (maxice8) [reply testing/moosefs/APKBUILD]: How we handle them ? poorly.

1. Add an $subpackage-openrc:_subpkg_openrc
2. write out the default_openrc() from abuild but with proper changes for that specific subpackage

Comment (uu) [reply testing/moosefs/APKBUILD]: Nailed it. Thank you :+1:

**Pull request closed without merging**